template/moleculer.config.ts

```diff
@@ -1,50 +1,79 @@
- * 	via environment variables, use the `MOL_` prefix and double underscore `__` for nested properties in .env file.
- * 	For example, to set the cacher prefix to `MYCACHE`, you should declare an env var as
- *  `MOL_CACHER__OPTIONS__PREFIX=MYCACHE`.
+ *    To overwrite broker’s deeply nested default options, which are not presented in "moleculer.config.js",
+ *    use the `MOL_` prefix and double underscore `__` for nested properties in .env file.
+ *    For example, to set the cacher prefix to `MYCACHE`, you should declare an env var as `MOL_CACHER__OPTIONS__PREFIX=mycache`.
+ *  It will set this:
+ *  {
+ *    cacher: {
+ *      options: {
+ *        prefix: "mycache"
+ *      }
+ *    }
+ *  }
  */
 const brokerConfig: BrokerOptions = {
 	// Namespace of nodes to segment your nodes on the same network.
 	namespace: "",
 	// Unique node identifier. Must be unique in a namespace.
 	nodeID: null,
+	// Custom metadata store. Store here what you want. Accessing: `this.broker.metadata`
+	metadata: {},
 
-	// Enable/disable logging or use custom logger. More info: https://moleculer.services/docs/0.13/logging.html
-	logger: true,
-	// Log level for built-in console logger. Available values: trace, debug, info, warn, error, fatal
+	// Enable/disable logging or use custom logger. More info: https://moleculer.services/docs/0.14/logging.html
+	// Available logger types: "Console", "File", "Pino", "Winston", "Bunyan", "debug", "Log4js", "Datadog"
+	logger: {
+		type: "Console",
+		options: {
+			// Using colors on the output
+			colors: true,
+			// Print module names with different colors (like docker-compose for containers)
+			moduleColors: false,
+			// Line formatter. It can be "json", "short", "simple", "full", a `Function` or a template string like "{timestamp} {level} {nodeID}/{mod}: {msg}"
+			formatter: "full",
+			// Custom object printer. If not defined, it uses the `util.inspect` method.
+			objectPrinter: null,
+			// Auto-padding the module name in order to messages begin at the same column.
+			autoPadding: false,
+		},
+	},
+	// Default log level for built-in console logger. It can be overwritten in logger options above.
+	// Available values: trace, debug, info, warn, error, fatal
 	logLevel: "info",
-	// Log formatter for built-in console logger. Available values: default, simple, short. It can be also a `Function`.
-	logFormatter: "default",
-	// Custom object & array printer for built-in console logger.
-	logObjectPrinter: null,
 
 	// Define transporter.
-	// More info: https://moleculer.services/docs/0.13/networking.html
-	transporter: "NATS",
+	// More info: https://moleculer.services/docs/0.14/networking.html
+	// Note: During the development, you don't need to define it because all services will be loaded locally.
+	// In production you can set it via `TRANSPORTER=nats://localhost:4222` environment variable.
+	transporter: "{{transporter}}",
```

Comment:
It should be null, as I mentioned in a previous PR. No need transporter for development time. In production, it will be set via env var. Copy the same part from the JS template.